MiniTravelBlog/MiniTravelBlog/PostViewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PostViewModel.swift
+//  MiniTravelBlog
+//
+//  Created by Charlotte Liang on 4/1/22.
+//
+
+import UIKit
+import FirebaseFirestoreSwift
+
+
+struct Post: Codable {
+  @DocumentID var id: String?
+  var description: String
+  var url: String
```

Comment:
I think it makes sense to make the type of this property a `URL`.
```suggestion
  var url: URL
```

Doing so avoid clients having to construct a URL from `url`:

```diff
AsyncImage(
-        url: URL(string:postViewModel.post.url),
+        url: postViewModel.post.url,
        content: { // ...
```
_from `MiniTravelBlog/MiniTravelBlog/ContentView.swift`_